### PR TITLE
Feature/packages/accordion list #cmx 126

### PR DESCRIPTION
**Pull request by Mar0cas**
Created new component accordion-list

### CODE REVIEW

File: packages/organisms/accordion-list/src/component/Defaults.ts [hunk 1]
```diff
@@ -0,0 +1,7 @@
+import { DefaultProps } from './Types';
+
+const Defaults: DefaultProps = {
+  title: 'Title',
```
Comment (ricardo-valente) [inline packages/organisms/accordion-list/src/component/Defaults.ts:4]: why `title` needs to have a default value?

File: packages/organisms/accordion-list/src/component/Utils.ts [hunk 2]
```diff
@@ -0,0 +1,2 @@
+import Utils from '@creat/common';
+export const notEmptyOrNull = (obj: any): boolean => Utils.notEmptyOrNull(obj);
```
Comment (ricardo-valente) [inline packages/organisms/accordion-list/src/component/Utils.ts:2]: why do we need to create an internal utils file just for importing and exporting a common util?

File: packages/organisms/accordion-list/src/__tests__/__snapshots__/AccordionList.test.tsx.snap [hunk 3]
```diff
@@ -0,0 +1,144 @@
...
+  data-layout="accordions"
+>
+  <div
+    class="container"
+  >
+    <div
+      class="row"
+    >
+      <div
+        class="wrapper wrapper--full wrapper--white -primary section__accordions-list"
```
Comment (ricardo-valente) [inline packages/organisms/accordion-list/src/__tests__/__snapshots__/AccordionList.test.tsx.snap:15]: wrapper-full does not exist

File: packages/organisms/accordion-list/src/component/Utils.ts (hunk 2 above)
Comment (Mar0cas) [reply packages/organisms/accordion-list/src/component/Utils.ts:2]: As explained on call:
Its do to consistency among other components and in the future we don't know if we need to create new functions.

File: packages/organisms/accordion-list/src/component/Defaults.ts (hunk 1 above)
Comment (Mar0cas) [reply packages/organisms/accordion-list/src/component/Defaults.ts:4]: Removed

File: packages/organisms/accordion-list/src/__tests__/__snapshots__/AccordionList.test.tsx.snap (hunk 3 above)
Comment (Mar0cas) [reply packages/organisms/accordion-list/src/__tests__/__snapshots__/AccordionList.test.tsx.snap:15]: We need to update the section component to fix this.

**Pull request merged**